starter/script.js

```diff
@@ -1 +1,35 @@
-// put code here!
+var buffer = function(){
+  console.log("*******************************")
+}
+
+console.log("You want to travel to a deserted island...")
+buffer()
+var pathInput = prompt("You can either take a plane or a helicopter..(Please enter 'p' or 'h')")
+
+if(pathInput == "p"){
+  console.log("You take the plane and everything is going well, in fact - you're ahead of scuedule. You hit some very bad turbulance and the plane crashes; you hit your head during the crash and it's all bloody now.")
+  buffer()
+  var responsePlaneCrash = prompt("You wounder what you want to do ... and decide to\n('b') - bandage yourself up with the left over medical supplies you could find in the cock pit.\n('w') - take a rest and wait until you feel better\n('d') - do nothing.")
+  if(responsePlaneCrash== 'b'){
```

Comment:
This is a stylistic thing, but be consistent with your spacing. It'll make your code more readable and more pleasing on the eyes (which'll be important when employers look through your code!).